api/baseline-api-v1.0.0.yaml

```diff
@@ -0,0 +1,4382 @@
+        nonce: '0x4fe1d5c3202230a8'
+        number: '0x9e1fd7'
+        parentHash: '0x279dbc9f7b7efe981ac58329bbca1864b39723df8aa362796a7a9bd5893ab1aa'
+        receiptsRoot: '0x7f874242eaa2e4c5b478f9e8b6abdd95cb10b939ad0fe000e8dc4807efacc4e6'
+        sha3Uncles: '0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347'
+        stateRoot: '0x76f111ed19e015cc7ce8d9a5e4f387365390adc5fcc97228705643e907eb793a'
+        timestamp: '0x60b840dc'
+        transactionsRoot: '0xed041ced1958b58c1040e1e09c456dcc551298eae8d7edca2659f6695a873f'
+    LoadBalancer:
+      title: Load Balancer
+      type: object
+      properties:
+        id:
+          type: string
+        created_at:
+          type: string
+        network_id:
+          type: string
+        workgroup_id:
+          type: string
+        name:
+          type: string
+        type:
+          type: string
+        host:
+          type: string
+        ipv4:
+          type: string
+          nullable: true
+        ipv6:
+          type: string
+          nullable: true
+        description:
+          type: string
+        region:
+          type: string
+        status:
+          type: string
+      example:
+        id: 64d209eb-a09d-4e2c-939e-4985a8aba70a
+        created_at: '2021-04-16T15:05:23.130Z'
+        network_id: 66d44f30-9092-4182-a3c4-bc02736d6ae5
+        workgroup_id: dc767890-0bbc-4c1d-bc04-f5842330018f
+        name: ceeb3bca-3b92-44b9-8ac5-fdd4564
+        type: provide
+        host: ceeb3bca-3b92-44b9-8ac5-fdd4564-717022042.us-east-2.elb.amazonaws.com
+        description: ceeb3bca-3b92-44b9-8ac5-fdd4564 - aws us-east-2
+        region: us-east-2
+        status: active
+        config:
+          api_port: 8080
+          container: providenetwork-ipfs
+          image: provide/ident
+          load_balancer_name: ceeb3bca-3b92-44b9-8ac5-fdd4564
+          load_balancer_url: ceeb3bca-3b92-44b9-8ac5-fdd4564-717022042.us-east-2.elb.amazonaws.com
+          provider_id: docker
+          region: us-east-2
+          role: peer
+          security:
+            egress: '*'
+            ingress:
+              0.0.0.0/0:
+                tcp:
+                  - 8080
+                udp: []
+          target_balancer_id: '''arn:aws:elasticloadbalancing:us-east-2:192663109025:loadbalancer/app/ceeb3bca-3b92-44b9-8ac5-fdd4564/8340a112fd942e75'''
+          target_id: aws
+          target_security_group_ids:
+            - sg-009a2c35c02c1ddc0
+          vpc_id: vpc-44df352f
+    LogMessage:
+      title: Log Message
+      type: object
+      description: is a format to receive system of record log messages in the proxy
+      properties:
+        baseline_id:
+          type: string
+        message:
+          type: string
+        object_id:
+          type: string
+        severity:
+          type: string
+        timestamp:
+          type: string
+          readOnly: true
+        type:
+          type: string
+    Message:
```

Comment:
need more context here. we need an ordering parameter that is deterministic and not the message ID. also where is sender/receiver? descriptions of properties? examples? payload is a string? not an object?